tests/unit/test.config.js

```diff
@@ -657,8 +657,8 @@ describe('config', () => {
       };
       assert.throws(() => {
         applyConf({...params, configObject});
-      }, WebExtError,
-        'WebExtError: Option: apiUrl was defined without a type.');
+      }, UsageError,
+        'UsageError: Option: apiUrl was defined without a type.');
     });
 
     it('throws an error when the type of one of them is in config' +
```

Comment:
@saintsebastian This test seems basically the same of the above test, I see that you have only changed the expected error here, but given that you looked into this test recently I'm wondering if you could check if I'm right and this test case is redundant.